src/components/NpiLevelTypography.tsx

```diff
@@ -15,16 +15,24 @@ const ScientificValueMapper: ScientificValueMapperType = {
 
 interface NpiLevelTypographyProps extends TypographyProps {
   scientificValue?: ScientificValue;
+  publisherId?: string;
 }
 
-export const NpiLevelTypography = ({ scientificValue, ...typographyProps }: NpiLevelTypographyProps) => {
+export const NpiLevelTypography = ({ scientificValue, publisherId, ...typographyProps }: NpiLevelTypographyProps) => {
   const { t } = useTranslation();
 
   const levelString = scientificValue ? ScientificValueMapper[scientificValue] : null;
+  const year = publisherId
+    ?.split('/')
+    .pop()
+    ?.match(/^\d{4}$/)
+    ? ` (${publisherId.split('/').pop()}) `
+    : '';
```

Comment:
Skjer veldig mye her, og jeg sliter iaf med å henge med. Enig i at dette er enklere?
```suggestion
  const year = publisherId?.split('/').pop();
  const yearString = year?.match(/^\d{4}$/) ? ` (${year})` : '';
```
Så kan vi bruke `yearString` på linje 35